### PR TITLE
run python api in eager model and filter the out in argument list

### PR DESCRIPTION
**Pull request by 2742195759**
### PR types

Bug fixes
### PR changes

Others

### Describe

run python api in eager model and filter the out in argument list

### CODE REVIEW

File: paddle/fluid/pybind/eager_utils.cc [hunk 1]
```diff
@@ -34,6 +34,7 @@ limitations under the License. */
 namespace paddle {
 namespace pybind {
 
+using PyNameVarBaseMap = std::unordered_map<std::string, py::handle>;
```
Comment (JiabinYang) [inline paddle/fluid/pybind/eager_utils.cc]: bad name, it seems not VarBaseMap

File: paddle/fluid/pybind/imperative.cc [hunk 2]
```diff
@@ -2079,8 +2103,8 @@ void BindImperative(py::module *m_ptr) {
               const PyNameVarBaseMap &ins, const PyNameVarBaseMap &outs,
               framework::AttributeMap attrs) {
              // TODO(xiongkun): move this function outside of tracer.
-             auto ins_map = ConvertToNameVarBaseMap(ins);
-             auto outs_map = ConvertToNameVarBaseMap(outs);
+             auto ins_map = ConvertToNameTensorMap(ins);
```
Comment (JiabinYang) [inline paddle/fluid/pybind/imperative.cc:2106]: what if its still VarBaseMap?

**Review by JiabinYang — COMMENTED** [review]
some comments

File: paddle/fluid/pybind/eager_utils.cc (hunk 1 above)
Comment (2742195759) [reply paddle/fluid/pybind/eager_utils.cc]: This name is not used, will be removed.

**Review by chenwhql — APPROVED** [review]
LGTM for PADDLE_ENFORCE

**Pull request merged**